src/imgui_graphics_settings.hpp

```diff
@@ -0,0 +1,97 @@
+#pragma once
+
+#include "render_tasks/d3d12_rtao_task.hpp"
+#include "render_tasks/d3d12_hbao.hpp"
+#include "render_tasks/d3d12_ansel.hpp"
+#include "render_tasks/d3d12_build_acceleration_structures.hpp"
+
+namespace wr::imgui::window
+{
+
+	static bool graphics_settings_open = true;
+
+
+	static wr::RTAOSettings rtao_user_settings;
+	static wr::HBAOSettings hbao_user_settings;
+	static wr::AnselSettings ansel_user_settings;
+	static wr::ASBuildSettings as_build_user_settings;
+
+	void GraphicsSettings(wr::FrameGraph* frame_graph)
+	{
+		if (graphics_settings_open)
```

Comment:
I prefer 1 window for every task. That would be easier to order when using the demo.
you could just close the tasks you don't use.

Reply:
done